Modules/Sources/WordPressUI/Extensions/Gravatar/Gravatar.swift

```diff
@@ -1,148 +0,0 @@
-public enum GravatarDefaultImage: String {
-    case fileNotFound = "404"
-    case mp
-    case identicon
-}
-
-@available(*, deprecated, message: "Use `AvatarURL` from the Gravatar iOS SDK. See: https://github.com/Automattic/Gravatar-SDK-iOS")
-public struct Gravatar {
-    fileprivate struct Defaults {
-        static let scheme = "https"
-        static let host = "secure.gravatar.com"
-        static let unknownHash = "ad516503a11cd5ca435acc9bb6523536"
-        static let baseURL = "https://gravatar.com/avatar"
-        static let imageSize = 80
-    }
-
-    public let canonicalURL: URL
-
-    public func urlWithSize(_ size: Int, defaultImage: GravatarDefaultImage? = nil) -> URL {
-        var components = URLComponents(url: canonicalURL, resolvingAgainstBaseURL: false)!
-        components.query = "s=\(size)&d=\(defaultImage?.rawValue ?? GravatarDefaultImage.fileNotFound.rawValue)"
-        return components.url!
-    }
-
-    public static func isGravatarURL(_ url: URL) -> Bool {
-        guard let components = URLComponents(url: url, resolvingAgainstBaseURL: false) else {
-            return false
-        }
-
-        guard let host = components.host, host.hasSuffix(".gravatar.com") else {
-                return false
-        }
-
-        guard url.path.hasPrefix("/avatar/") else {
-                return false
-        }
-
-        return true
-    }
-
-    /// Returns the Gravatar URL, for a given email, with the specified size + rating.
-    ///
-    /// - Parameters:
-    ///     - email: the user's email
-    ///     - size: required download size
-    ///     - rating: image rating filtering
-    ///
-    /// - Returns: Gravatar's URL
-    ///
-    public static func gravatarUrl(for email: String,
-                                   defaultImage: GravatarDefaultImage? = nil,
-                                   size: Int? = nil,
-                                   rating: GravatarRatings = .default) -> URL? {
-        let hash = gravatarHash(of: email)
-        let targetURL = String(format: "%@/%@?d=%@&s=%d&r=%@",
-                               Defaults.baseURL,
-                               hash,
-                               defaultImage?.rawValue ?? GravatarDefaultImage.fileNotFound.rawValue,
-                               size ?? Defaults.imageSize,
-                               rating.stringValue())
-        return URL(string: targetURL)
-    }
-
-    /// Returns the gravatar hash of an email
-    ///
-    /// - Parameter email: the email associated with the gravatar
-    /// - Returns: hashed email
-    ///
-    /// This really ought to be in a different place, like Gravatar.swift, but there's
-    /// lots of duplication around gravatars -nh
-    private static func gravatarHash(of email: String) -> String {
-        return email
-            .lowercased()
-            .trimmingCharacters(in: .whitespaces)
-            .sha256Hash()
```

Comment:
(Nit) `sha256Hash` can now also be removed. I left a note to do it later unless you get to it first.

Reply:
Done 👍